### ISSUE TITLE
Handle Button disable for both appearance and functionality

### ISSUE THREAD

**Issue by elliemaynard** [gumgum/gumdrops#89]
Right now, when we add a Button to a page, we have to specify twice that we want a button to be disabled: both visually, and so that it can't be clicked:

```
<Button
  context="success"
  size="xs"
  name="save"
  onClick={handleSubmit}
  className={cx({
     '-disabled': isSubmitDisabled,
  })}
  disabled={isSubmitDisabled}
>
  Add New Pixel
</Button>
```
I'm not thinking of a reason why we would want to do it this way, where a button would be disabled visually but still be able to be clicked? Could we have it where the disabled prop handles both things?

**Comment by edrpls** [comment]
Thanks @elliemaynard! Just ran into this issue, I'll patch it today!

**Comment by edrpls** [comment]
Fixed in #91
